wireshark/README.md

```diff
@@ -60,50 +46,25 @@ $ brew install wireshark
 $ sudo apt install wireshark-dev
 ```
 
-## Compile the dissector
-
-> **Tip**
-> 
-> If the compiler cannot find the Wireshark headers, add the include path manually.
-> `-DWIRESHARK_INCLUDE_PATH=<WIRESHARK_INCLUDE_PATH>`
+### Build from source
 
-Compile the dissector.
+Run the following commands in this subdirectory.
 
 ```shell
-cmake -DBUILD_WIRESHARK=ON .
-make pulsar-dissector
+cmake -B build
+cmake --build build
 ```
 
-This creates the `pulsar-dissector.so` plugin library in the Wireshark directory.
+Then the `pulsar-dissector.so` plugin will be created under the `build-wireshark` directory.
 
-## Install Wireshark dissector
+### Copy to the plugin directory
 
-Copy the dissector in the appropriate location so that Wireshark can find it at startup.
+Run the Wireshark, then click the menu `Help - About Wireshark - Plugins`, you will find the plugin directory. For example, it's `/usr/lib/x86_64-linux-gnu/wireshark/plugins/3.2/epan` on Ubuntu 20.04. Then, copy the dissector into that directory:
```

Comment:
```suggestion
1. Run the Wireshark and click **Help - About Wireshark - Plugins**, and then you can find the plugin directory. For example, it's `/usr/lib/x86_64-linux-gnu/wireshark/plugins/3.2/epan` on Ubuntu 20.04. 

2. Copy the dissector into that directory.

```
In technical writing, we usually use bold for menus. Details see https://docs.google.com/document/d/1lc5j4RtuLIzlEYCBo97AC8-U_3Erzs_lxpkDuseU0n4/edit?pli=1#bookmark=kix.m5ccxnsf33mu